packages/core/src/payment/strategies/stripe-upe/stripe-upe-payment-strategy.ts

```diff
@@ -238,6 +238,14 @@ export default class StripeUPEPaymentStrategy implements PaymentStrategy {
         }
     }
 
+    private _getChannelName() {
+        const channel_name =
+            this._store.getState().config.getConfig()?.storeConfig?.links.siteLink ||
+            'problem getting channel name';
```

Comment:
This seems a bit odd that if we don't have a `siteLink` then we return `problem getting channel name` as a channel name 🤔